pyproject.toml

```diff
@@ -17,21 +17,15 @@ dependencies = [
     "torch>=2.8.0",
     "uvicorn>=0.36.0",
     "wandb>=0.21.3",
+    "rustbpe==0.1.0"
```

Comment:
If you plan on publishing both packages to PyPI, this bound may need to be adjusted.